extensions/dacpac/package.json

```diff
@@ -37,6 +37,16 @@
 					"command": "dacFx.start",
 					"when": "connectionProvider == MSSQL && nodeType && nodeType == Database",
 					"group": "export"
+				},
+				{
+					"command": "dacFx.start",
+					"when": "connectionProvider == MSSQL && nodeType && nodeType == Server",
+					"group": "export"
+				},
+				{
+					"command": "dacFx.start",
+					"when": "connectionProvider == MSSQL && nodeType && nodeType == Folder && nodeLabel == 'Databases'",
+					"group": "export"
```

Comment:
One more clarification - I see that in Wizard.start(), we use Profile.dbname for getting the db - please double check that it will return valid values once we do this from folder/server.

Reply:
Yes there's no problem. If the wizard is launched from the database folder or server, then the server value default is correct and the default database is whatever is the first database returned from azdata.connection.listDatabases()